content/servicemesh/deploy.md

```diff
@@ -0,0 +1,84 @@
+---
+title: "Deploy Sample Apps"
+date: 2018-11-13T16:37:17+09:00
+weight: 40
+draft: false
+---
+
+> Now that we have all the resources installed for Istio, we will use sample application called BookInfo to review key capabilities of Service Mesh such as intelligent routing and review telemetry data using Prometheus & Grafana.
+
+### Sample Apps
+
+
+![Sample Apps](/images/servicemesh-deploy1.png)
+
+The Bookinfo application is broken into four separate microservices:
+
+* <span style="color:orange">**productpage**</span>
+  * The productpage microservice calls the details and reviews microservices to populate the page.
+
+* <span style="color:orange">**details**</span>
+  * The details microservice contains book information.
+
+* <span style="color:orange">**reviews**</span>
+  * The reviews microservice contains book reviews. It also calls the ratings microservice.
+
+* <span style="color:orange">**ratings**</span>
+  * The ratings microservice contains book ranking information that accompanies a book review.
+
+There are 3 versions of the <span style="color:orange">*reviews*</span> microservice:
+
+* Version v1
+  * doesn’t call the ratings service.
+
+* Version v2
+  * calls the ratings service, and displays each rating as 1 to 5 <span style="color:black">**black stars**</span>.
+
+* Version v3
+  * calls the ratings service, and displays each rating as 1 to 5 <span style="color:red">**red stars**</span>.
+
+---
+
+### Deploy Sample Apps
+
+Deploy sample apps by manually injecting istio proxy and confirm pods, services are running correctly
+
+```
+kubectl apply -f <(istioctl kube-inject -f samples/bookinfo/platform/kube/bookinfo.yaml)
```

Comment:
let's break the command apart from the sample output here.

Reply:
agree. we should keep Copy commands separate from Output. Its okay to have two separate blocks to represent this

Reply:
break the output to the another windows apart from command